### ISSUE TITLE
Full screen

### ISSUE THREAD

**Issue by amrboxit4me** [PierfrancescoSoffritti/android-youtube-player#205]
I understand the full screen from the sample just I suggest you to add this note
that full screen need a full view or frame layout
so they could use this     otherlayout.setVisibility(View.GONE); on Entering full screen mode

**Comment by PierfrancescoSoffritti** [comment]
Hi, sorry I couldn't understand. Could you rephrase?

**Comment by amrboxit4me** [comment]
Ok if I made full screen while there are other views in the same layout it doesn't work so I hide other views in the same layout

```
private void addFullScreenListenerToPlayer() {
        youTubePlayerView.addFullScreenListener(new YouTubePlayerFullScreenListener() {
            @Override
            public void onYouTubePlayerEnterFullScreen() {

                setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_LANDSCAPE);
                fullScreenHelper.enterFullScreen(youTubePlayerView);

                ivLogo.setVisibility(View.GONE);
                cardSignAs.setVisibility(View.GONE);
                cardVideo.getLayoutParams().height = LinearLayout.LayoutParams.MATCH_PARENT;
            }

            @Override
            public void onYouTubePlayerExitFullScreen() {
                cardSignAs.setVisibility(View.VISIBLE);
                ivLogo.setVisibility(View.VISIBLE);
                cardVideo.getLayoutParams().height = 0;


                setRequestedOrientation(ActivityInfo.SCREEN_ORIENTATION_PORTRAIT);
                fullScreenHelper.exitFullScreen(youTubePlayerView);
            }
        });
    }
```

**Comment by amrboxit4me** [comment]
Then on exit full screen, I revert things back so I put `cardVideo.getLayoutParams().height = 0;`
so Linear weight works again

**Comment by PierfrancescoSoffritti** [comment]
I see. This works in your specific use case, not everyone will need it. For example my sample activity doesn't need it because the container of the `YouTubePlayerView` is already full screen. Therefore I don't think this should be added.
